stage0/supply-chain/config.toml

```diff
@@ -0,0 +1,52 @@
+# cargo-vet config file
+[cargo-vet]
+version = "0.6"
+
+[imports.google]
+url = "https://raw.githubusercontent.com/google/supply-chain/main/audits.toml"
+
+[imports.google.criteria-map]
+crypto-safe = "crypto-safe"
+does-not-implement-crypto = "does-not-implement-crypto"
+
+[imports.mozilla]
+url = "https://raw.githubusercontent.com/mozilla/supply-chain/main/audits.toml"
+
+[policy.bitflags]
+criteria = ["safe-to-deploy", "crypto-safe"]
+
+[policy.elf]
+criteria = ["safe-to-deploy", "crypto-safe"]
+
+[policy.log]
+criteria = ["safe-to-deploy", "crypto-safe"]
```

Comment:
I understand that `does-not-implement-crypto` implies `crypto-safe`, but wouldn't it be more explicit (and specific) to require `log` (and most other crates here) to actually have `does-not-implement-crypto`?

Reply:
Fundamentally, yes, I'd really be surprised if `log` wants to implement crypto :)

What I was trying to capture here is that the default should be `["safe-to-deploy", "crypto-safe"]` but unfortunately there's no global setting (yet) to do that in cargo vet, so we need to specify this for each crate manually.

We can tighten (or loosen) the requirements as we wish later on.